src/smexperiments/trial_component.py

```diff
@@ -137,4 +150,7 @@ def list(cls, source_arn=None, created_before=None, created_after=None,
             created_after=created_after,
             sort_by=sort_by,
             sort_order=sort_order,
-            sagemaker_boto_client=sagemaker_boto_client)
+            sagemaker_boto_client=sagemaker_boto_client,
+            trial_name=trial_name,
```

Comment:
I do not see trial name from https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.list_trial_components, should we filter the trail name through the response from boto client?

Reply:
it hasn't been released yet, we will hold off on pushing this to pypi until then.